apps/innsendt-soeknad/src/main/kotlin/JournalpostSkrevet.kt

```diff
@@ -33,38 +33,53 @@ internal class JournalpostSkrevet(
         val dokumentInfoId = dokarkivRetur.path("dokumenter")[0]?.path("dokumentInfoId")?.asLong() ?: 0L
 
         val soeknadId = packet["@lagret_soeknad_id"]
-        if (soeknadId.toString().startsWith("TEST-") || soeknadId.asLong() == 0L) {
-            logger.info("Verifiseringssøknad med id $soeknadId lest med dokumentInfoId $dokumentInfoId")
-            return
-        }
+        val harTestSoeknad = soeknadId.toString().startsWith("TEST-") || soeknadId.asLong() == 0L
```

Comment:
Ser for meg ut som hvis `harTestSoeknad` så return så slipper man alle disse if elsene nedover som bare logger?

Reply:
Trenger vi de logginnslagene, kan man ikke ha de som de var?